tests/tmath.nim

```diff
@@ -10,6 +10,13 @@ cDebug()
 cDisableCaching()
 
 cAddStdDir()
+
+cPlugin:
+  import strutils
+
+  proc onSymbol*(sym: var Symbol) {.exportc, dynlib.} =
```

Comment:
this doesn't match `onSymbolType* = proc(sym: var Symbol) {.cdecl.}`
`{.cdecl.}` vs `{.exportc, dynlib.}`

Reply:
The plugin needs to export it, toast is pulling it in so has a different signature.